scripts/feedme.coffee

```diff
@@ -129,18 +130,18 @@ module.exports = (robot) ->
 
   # wait for given time and then send a message to #vie-food with all
   # subscribed users for this food type.
-  setReminder = (foodType, hour = FOOD_TIME.HOUR, minute = FOOD_TIME.MINUTE) ->
+  setReminder = (foodType, hour = FOOD_TIME.HOUR, minute = FOOD_TIME.MINUTE, offset = FOOD_TIME.OFFSET ) ->
     setTimeout ( =>
       eater = robot.brain.get "feedme.eater"
       people = if eater and foodType of eater then eater[foodType] else []
-      msg = if people.length > 0 then "#{people.toString()}:\n" else ""
+      msg = if people.length > 0 then "#{people.join(", ")}:\n" else ""
       robot.messageRoom "vie-food", msg + "Los los ... #{FOODTYPES[foodType]} wartet nicht!"
```

Comment:
Not related, but we're using a hard-coded channel name here, which is probably not so sweet.